travis-install-wx.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -e
+rm -Rf wxPython-src-2.8.12.1
+wget http://sourceforge.net/projects/wxpython/files/wxPython/2.8.12.1/wxPython-src-2.8.12.1.tar.bz2/download
```

Comment:
I have added a checksum check. However please note that since I could not find official checksums for the wxPython source zip I used the source zip that I downloaded locally. This will only protect against future changes of the file.